package/CHANGELOG

```diff
@@ -83,6 +83,7 @@ Fixes
   * Contact Analysis class respects PBC (Issue #2368)
 
 Enhancements
+  * Added computation of Mean Squared Displacements (PR #2619)
```

Comment:
Also reference the issue #2438 (that's generally more important than the PR number; you can include the PR if you think that one should also look at the PR for context). In this case I'd say
```suggestion
  * Added computation of Mean Squared Displacements (#2438, PR #2619)
```